applications/FluidDynamicsApplication/tests/AdjointQSVMSSensitivity2DTest/cylinder_test_materials.json

```diff
@@ -0,0 +1,19 @@
+{
+    "properties": [
+        {
+            "model_part_name": "MainModelPart.Parts_Fluid",
+            "properties_id": 1,
+            "Material": {
+                "constitutive_law": {
+                    "name": "Newtonian2DLaw"
+                },
+                "Variables": {
+                    "DENSITY": 1.0E+03,
+                    "VISCOSITY": 1.0E-06,
+                    "DYNAMIC_VISCOSITY": 1.0E-03
```

Comment:
Why providing both? I'd provide the only the one that is to be used to avoid misunderstandings.

Reply:
done :)